tyrant/cogs/lemon_say.py

```diff
@@ -0,0 +1,62 @@
+import logging
+import random
+
+from discord.ext import commands
+from discord.ext.commands import Bot, Cog, Context
+
+from tyrant import constants
+
+log = logging.getLogger(__name__)
+
+
+class LemonSay(Cog):
+    """Post answer to generic yes/no questions"""
```

Comment:
```suggestion
    """Provide answers for generic yes/no questions."""
```
Docstrings should end with punctuation. See [PEP 257](https://www.python.org/dev/peps/pep-0257/)